Sources/Web3Core/KeystoreManager/EthereumKeystoreV3.swift

```diff
@@ -23,13 +23,13 @@ public class EthereumKeystoreV3: AbstractKeystore {
     }
 
     public func UNSAFE_getPrivateKeyData(password: String, account: EthereumAddress) throws -> Data {
-        if self.addresses?.count == 1 && account == self.addresses?.last {
-            guard let privateKey = try? self.getKeyData(password) else {
+        if account == addresses?.last {
+            guard let privateKey = try? getKeyData(password) else {
```

Comment:
Maybe we can make this bit a bit more straightforward? Like changing the `if let` above to an another guard statement.

Reply:
Upd: oh, I see that the execution flow continues after that line:32 throw statement. Maybe we should add at least `TODO: refactor me` at the very beginning of the whole file, as I believe this function should be split in to, but not this time?